module/batch-action/src/Application/Form/BatchActionReprocessingFormInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Application\Form;
+
+interface BatchActionReprocessingFormInterface
```

Comment:
```suggestion
interface BatchActionReprocessFormInterface
```
More in line with abstraction and other classes